multiple-erc20-to-erc20/contracts/contracts/upgradeable_contracts/factories/HomeBridgeFactory.sol

```diff
@@ -89,7 +89,7 @@ contract HomeBridgeFactory is BasicBridgeFactory {
         // cast proxy as IHomeBridge
         IHomeBridge homeBridge = IHomeBridge(proxy);
         // initialize homeBridge
-        homeBridge.initialize(bridgeValidators, homeDailyLimit(), homeMaxPerTx(), minPerTx(), gasPrice(), requiredBlockConfirmations(), token, foreignDailyLimit(), foreignMaxPerTx(), homeBridgeOwner());
+        homeBridge.initialize(bridgeValidators, adjustToDefaultDecimals(homeDailyLimit(), _tokenDecimals), adjustToDefaultDecimals(homeMaxPerTx(), _tokenDecimals), adjustToDefaultDecimals(minPerTx(), _tokenDecimals), gasPrice(), requiredBlockConfirmations(), token, adjustToDefaultDecimals(foreignDailyLimit(), _tokenDecimals), adjustToDefaultDecimals(foreignMaxPerTx(), _tokenDecimals), homeBridgeOwner());
```

Comment:
@LiorRabin so it's stuck here. Actually one less use of `adjustToDefaultDecimals` and it passing.
Not sure why we need both foreign and home limits here